### PR TITLE
[READY] Incinerator clean up and update

### PR DESCRIPTION
**Pull request by granpawalton**
:cl:
tweak: Removed old piping sections and replaced with Canister storage area in atmos incinerator
tweak: scrubber and distro pipes moved in atmos incinerator to make room for added piping
balance: added filter at connector on scrubbing pipe in atmos incinerator
balance: replaced vent in incinerator with scrubber in **Both** incinerators
balance: mixer placed on pure loop at plasma
fix: delta and pubby atmos incinerator air alarm is no longer locked at round start
fix: pubby atmos incinerator now starts without atmos in it
/:cl:

Main goal: clean up atmos so it is more user friendly and up to date with its current uses
Side goal: provide the backbone of making tritium for the user while still requiring several edits to make it at a rate that can be used.
Reasoning: atmos can take a lot of math and a lot of trivial work to make it reach its end teir developements, so much so that many, especially newer atmos techs arent able to make anything tangible until around the 30 minute mark which is the length of most rounds that arent on low pop hours. When it comes to more advanced incinerators users the filter, scrubber, and mixer are all mandatory changes for a tritium set up which can become very boring and just function as a time gate between the start of the round and tritium to work with. These changes are meant to remove the less thought requiring parts of incinerator changes and encourage newer players to make their first exotic gas (similiar to how each department has a simple entry level job that a player can get hooked on to start). The changing in scrubbers and distro are meant to allow players to make many common changes without removing parts the scrubbers and distro from the room itself and the removing of the old piping for the same reason. Removing the old piping will also allow the room to be less cluttered after the incinerator user makes his or her changes. Toxins incinerator was also updated for universality

My concerns: Some of the changes to create the backbone of tritium production (especially the mixer on pure) i am not 100% for and would very much appreciate feedback on. The scrubber change to incinerator would mean using a hardsuit isnt _required_ for making tritium but is for many optimizations would could be disencouraging to actually optimize. Lastly im not worried about new max cap rushes that could occur as to make it actually quickly with trit you would need all access but what i am worried about is max cap rushes to generate research point being sped up by this, it would take a competent player as many dont know much about toxins or atmos but current research bomb rushes are pretty fast without the tritium backbone changes. Feedback is highly appreciated.

meta changes: https://imgur.com/a/xvjgBPO
box changes: https://imgur.com/a/q4ZkckH
pubby changes: https://imgur.com/a/RNv2rO1
delta changes: https://imgur.com/a/M70ryV2
plasma mixer example: https://imgur.com/a/ArQgTYf

Edit: less tags
Edit: removed WIP

### CODE REVIEW

**Comment by granpawalton** [comment]
also im not sure why other peoples mapping changes are in my diff and if anyone else knows about what could cause that that would be appreciated

**Comment by Anonmare** [comment]
I honestly really hate those snowflake vents that suck instead of blow

**Comment by ExcessiveUseOfCobblestone** [comment]
This is an alright change, just hate the round-time rationale.

The more features that are long term, the more people have investment into the round which will consequently prolong them.

**Comment by Kagus** [comment]
Heck, even just including a mixer straight off the bat is a big deal, as the default piping setup is useless for any purpose, including the rarity of trying to actually make power via the turbine. Without the necessary control to define an actual burn mix, the chamber can expect to reach a nice hot temperature of "diarrhea".

Any intentions of adding a mixer or an RPD to the pirate cutter? I know it's kind of a meaningless change to a rare and meaningless event, but god damn the piping for that engine makes me feel bad things.


Of course, a proper rebalancing of the turbines themselves so you can't just make free energy from spamming them in the hallways would be delightful as well, but really, I'm happy just with less pointless busywork fixing the pipes at roundstart.

...I mean, yes, I appreciate that basically the entirety of playing an atmos tech is "fixing the pipes at roundstart", but you get what I mean.

**Comment by granpawalton** [comment]
@Kagus yeah i can add a RPD to the pirate ship event as i honestly dont think they are gonna be able to do anything bigger with it to the crew they couldnt already before. Also you reminded me to check for other possible incinerators besides station z-level so thanks for that

File: _maps/shuttles/ruin_syndicate_dropship.dmm [hunk 1]
```diff
@@ -630,7 +630,7 @@ Tn
 Lq
 YU
 Dx
-zq
+ZI
```
Comment (granpawalton) [inline _maps/shuttles/ruin_syndicate_dropship.dmm]: is this an issue? i don't know why it renamed it but it seems to be the same tile.

**Comment by granpawalton** [comment]
edits box science incinerator so will have minor conflict with #39318

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 2]
```diff
@@ -2,10 +2,30 @@
 "aaa" = (
 /turf/open/space/basic,
 /area/space)
+"aab" = (
+/obj/machinery/atmospherics/components/trinary/mixer{
+	dir = 1;
+	icon_state = "mixer_off";
```
Comment (ShizCalev) [inline _maps/map_files/MetaStation/MetaStation.dmm]: dv

File: _maps/map_files/PubbyStation/PubbyStation.dmm [hunk 3]
```diff
@@ -37231,12 +37231,14 @@
...
 "bSg" = (
-/obj/machinery/atmospherics/pipe/manifold/yellow/visible{
-	dir = 8
-	},
 /obj/structure/cable{
 	icon_state = "1-2"
 	},
+/obj/machinery/atmospherics/components/trinary/mixer{
+	name = "plasma mixer";
+	icon_state = "mixer_off";
```
Comment (ShizCalev) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: dv

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: _maps/shuttles/ruin_syndicate_dropship.dmm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply _maps/shuttles/ruin_syndicate_dropship.dmm]: no

**Pull request merged**